### PR TITLE
Fix spans for cached items at the manager level, not the service level

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Followup to https://github.com/dotnet/roslyn/pull/53259

### CODE REVIEW

File: src/Features/Core/Portable/Completion/CompletionService.cs [hunk 1]
```diff
@@ -171,24 +171,6 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
...
 
-        /// <summary>
-        /// Preferred overload of <see cref="GetChangeAsync(Document, CompletionItem, char?,
-        /// CancellationToken)"/>.
-        ///
-        /// This overload is passed the value of <see cref="CompletionContext.CompletionListSpan"/>
-        /// which should be used to determine what span should be updated in the original <paramref
-        /// name="document"/> passed in.
-        /// </summary>
-        internal virtual Task<CompletionChange> GetChangeAsync(
```
Comment (333fred) [inline src/Features/Core/Portable/Completion/CompletionService.cs:182]: You'll need to remove the exposed member in the OmniSharp external access as well.

Comment (CyrusNajmabadi) [reply src/Features/Core/Portable/Completion/CompletionService.cs:182]: sgtm

File: src/Tools/ExternalAccess/OmniSharp/Completion/OmniSharpCompletionService.cs [hunk 2]
```diff
@@ -31,7 +31,7 @@ internal static class OmniSharpCompletionService
             char? commitCharacter = null,
             bool disallowAddingImports = false,
             CancellationToken cancellationToken = default)
-            => completionService.GetChangeAsync(document, item, completionListSpan, commitCharacter, disallowAddingImports, cancellationToken);
+            => completionService.GetChangeAsync(document, item, commitCharacter, cancellationToken);
```
Comment (CyrusNajmabadi) [inline src/Tools/ExternalAccess/OmniSharp/Completion/OmniSharpCompletionService.cs:34]: @333fred do you want me to leave the unused parameters?  or should i remove those as well?

Comment (333fred) [reply src/Tools/ExternalAccess/OmniSharp/Completion/OmniSharpCompletionService.cs:34]: Just remove the method entirely. We'll move to the public API.

**Review by 333fred — APPROVED** [review]
OmniSharp change LGTM

**Comment by 333fred** [comment]
@CyrusNajmabadi sorry to disable auto-merge, but can you please take this for 16.11 or 16.10 instead?

**Comment by CyrusNajmabadi** [comment]
Still auto-merging.  We can make anotehr PR to backport :)

**Pull request merged**